webanno-ui-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/curation/component/model/SuggestionBuilder.java

```diff
@@ -334,8 +343,9 @@ public CAS getMergeCas(AnnotatorState aState, SourceDocument aDocument,
         }
         // Create JCas, if it could not be loaded from the file system
         catch (Exception e) {
-            return initializeMergeCas(aState, aCasses, aTemplate, aMergeIncompleteAnnotations);
+            log.error(e.getMessage());
```

Comment:
I think we should probably not catch exception here but rather let the exception bubble up if there is any.